.github/workflows/build.yaml

```diff
@@ -41,94 +43,31 @@ jobs:
         env:
           PKG_CONFIG_PATH: /usr/lib/x86_64-linux-gnu/pkgconfig:/opt/bb/lib64/pkgconfig
         run: |
-          cmake -S . -B cmake.bld/Linux -G Ninja \
+          cmake -S . -B build/blazingmq -G Ninja \
             -DCMAKE_TOOLCHAIN_FILE=${{ github.workspace }}/deps/srcs/bde-tools/BdeBuildSystem/toolchains/linux/gcc-default.cmake \
             -DCMAKE_BUILD_TYPE=Debug \
             -DBDE_BUILD_TARGET_SAFE=ON \
             -DBDE_BUILD_TARGET_64=ON \
             -DBDE_BUILD_TARGET_CPP17=ON \
             -DCMAKE_PREFIX_PATH=${{ github.workspace }}/deps/srcs/bde-tools/BdeBuildSystem \
             -DCMAKE_INSTALL_LIBDIR=lib64
-          cmake --build cmake.bld/Linux --parallel 8 --target all all.t
+          cmake --build build/blazingmq --parallel 8 --target all all.t
       - name: Run Unit Tests
         run: |
-          cd cmake.bld/Linux
+          cd build/blazingmq
           ctest -E mwcsys_executil.t --output-on-failure
-      - uses: actions/cache@v3
-        with:
-          path: |
-            cmake.bld/Linux
-            deps
-            /opt/bb/include
-          key: cache-${{ github.sha }}
-
-  build_and_test_bmqprometheus_plugin:
```

Comment:
Any reason to remove this hook?

Reply:
Nope. I don't even recall doing this. Restoring...